### PR TITLE
Makes PERSON_ANIMATION the default attack animation setting.

### PR DESCRIPTION
**Pull request by PJB3005**

### CODE REVIEW

**Comment by PJB3005** [comment]
Actually I probably need to make a migration too, damnit.

**Comment by brnor** [comment]
Do we even bother throwing around opinions on this? Here's mine:
I don't personally like the person animation simply because too often it looks like someone's having a spasm attack, especially if multiple people are mashing attack on each other. It's just stupid-looking.
I don't know how else it would be changed to look acceptable, I can say that this version that's currently up is definitely better than what I'm used to seeing, others always tended too be too long and dragged out. Even with this shorter version, oftentimes some attacks get their animation skipped because the attack happened too quick. For some reason I also feel like it looks **too** smooth.

The item_animation has the problem, in my opinion, being way too large(especially the fists). Again, this version is much better than the first one where the animation always played from uptop and descended down on the character, but could still use some work I guess. Someone in thread also suggested changing the animation icons (if I understood correctly, that person wanted the icons changed to the icons which are shown on the body, not in the hand display in the UI. Not sure if I got this right, this might even look worse?).

In summary, right now I'm in preference of the item_animations, which used to be the default when this was implemented, but they'd need some tweaking to the look.

also

- 173247220
kill yourself

**Comment by Iamgoofball** [comment]
why not both item and person

**Comment by Iamgoofball** [comment]
also re: making it use the onmob

we tried that at /tg/ for awhile and it was so hard to see it was practically useless

also looked funky because offsets

**Comment by Shadowmech88** [comment]
Personally I like PERSON_ANIMATION better because it's less immulsion-breaking than a giant hologram of the item you're holding flying toward your target.

That's just my opinion, though.

**Comment by PJB3005** [comment]
Yep, what @Shadowmech88 said is why I changed it to this, though I'm fine with changing it to item.

**Comment by PJB3005** [comment]
afbeelding

F
